cms/pytest.ini

```diff
@@ -1,6 +1,9 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = cms.envs.test
 addopts = --nomigrations --reuse-db --durations=20 -p no:randomly
+filterwarnings =
+    default
```

Comment:
What does this `default` mean? What does it match to filter?

Reply:
Oh - just read the removed comment in test.py. I assume it does the same here as described?
Nit: Perhaps preserve that comment somewhere in this config file?